.jenkins

```diff
@@ -21,13 +22,17 @@ pipeline {
                     }
                     steps {
                         sh 'ccache --zero-stats'
```

Comment:
Do you still want to keep this?

Reply:
Yes.  We only want the stats for that build.

Reply:
That said on a slave machine that allow concurrent builds the stat could be garbage